testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/hibernate/SFSBHibernate2LcacheStats.java

```diff
@@ -88,6 +88,7 @@ public void setupConfig() {
             configuration.setProperty(Environment.DATASOURCE, "java:jboss/datasources/ExampleDS");
             // set property to enable statistics
             configuration.setProperty("hibernate.generate_statistics", "true");
+            configuration.getProperties().put(AvailableSettings.TRANSACTION_COORDINATOR_STRATEGY, "jta");
```

Comment:
This change helped but we also need the same line added to org.jboss.as.test.integration.hibernate.secondlevelcache.SFSB, after line 82.  This helped with my local testing.